modules/nvidia_plugin/README.md

```diff
@@ -9,7 +9,7 @@ OpenVINO™ NVIDIA GPU plugin is supported and validated on the following platfo
 
 OS                     | GPU
 ---------------------- | ----------------------
-Ubuntu* 18.04 (64-bit) | Geforce 1080 Ti, NVIDIA T4
+Ubuntu* 20.04 (64-bit) | Geforce 1080 Ti, NVIDIA T4
```

Comment:
Are list of devices still the same?

Reply:
Quadro RTX 4000 is what we have now

Reply:
> Quadro RTX 4000 is what we have now

Maybe there is a sense to update it as well?